src/cuga/sdk.py

```
@@ -538,6 +539,174 @@ async def add_tool_guide(
+        )
+
+        # Update in storage
+        await policy_system.storage.update_policy(updated_policy)
+        await policy_system.initialize()  # Reload policies
+
+        # Save to filesystem if sync is enabled
+        if self._fs_sync:
+            try:
+                self._fs_sync.save_policy_to_file(updated_policy)
+                logger.debug(f"Saved updated policy '{policy_id}' to filesystem")
+            except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)